src/SFA.DAS.DATA.MANAGEMENT/Tables/AsDataPL_RP_FinancialReview.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [AsData_PL].[RP_FinancialReview]
+(
+	[Id]									[uniqueidentifier]			NOT NULL,
+	[ApplicationId]							[uniqueidentifier]			NOT NULL,
+	[Status]								[nvarchar](20)				NOT NULL,
+	[SelectedGrade]							[nvarchar](20)				NULL,
+	[FinancialDueDate]						[datetime2](7)				NULL,
+	[GradedBy]								[nvarchar](256)				NULL,
+	[GradedOn]								[datetime2](7)				NULL,
+	[Comments]								[nvarchar](max)				NULL,
+	[ExternalComments]						[nvarchar](max)				NULL,	
+	[ClarificationRequestedOn]				[datetime2](7)				NULL,
+	[ClarificationRequestedBy]				[nvarchar](256)				NULL,
```

Comment:
clarificationrequestedby - hope this is only a guid

Reply:
This is free text not a GUID data it seems and loading it as a masked column